Makefile

```diff
@@ -69,9 +71,8 @@ bin/subctl-%: pkg/subctl/operator/common/embeddedyamls/yamls.go $(shell find pkg
 	GOOS=$${components[-2]}; \
 	GOARCH=$${components[-1]}; \
 	export GOARCH GOOS; \
-	source $(SCRIPTS_DIR)/lib/version; \
 	$(SCRIPTS_DIR)/compile.sh \
-		--ldflags "-X github.com/submariner-io/submariner-operator/pkg/version.Version=$${VERSION}" \
+		--ldflags "-X github.com/submariner-io/submariner-operator/pkg/version.Version=$(VERSION)" \
```

Comment:
Wait, you are reverting the changes that generate subctl-devel- binaries for bleeding edge.

Reply:
Please see: https://github.com/submariner-io/submariner-operator/pull/522/files

Reply:
Good catch, thanks! I’ve updated the patch to use more explicit variable names in this instance.